blazor/linear-gauge/accessibility.md

```diff
@@ -38,7 +37,7 @@ The accessibility compliance for the Linear Gauge component is outlined below.
 
 ## WAI-ARIA attributes
 
-The Linear Gauge component followed the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following ARIA attributes are used in the Linear Gauge component.
+The Blazor Linear Gauge component followed the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following `WAI-ARIA` attributes are used in the Linear Gauge component.
```

Comment:
In one component, WAI-ARIA is linked in the second line. 
![chrome_kFADC5HJuV](https://github.com/syncfusion-content/blazor-docs/assets/93366915/0a095b65-b7ff-45a7-bc9d-15c3c461158d)

But here code style is provided. Which one we should follow?

Reply:
Hi @sabari-senthamaraikannan ,

I modified the reference link.

Regards,
@Backiaraj